tests/src/main/java/com/netflix/msl/util/MslTestUtils.java

```diff
@@ -238,9 +238,10 @@ public static UserIdToken getUntrustedUserIdToken(final MslContext ctx, final Ma
         final ICryptoContext cryptoContext = new NullCryptoContext();
         final Set<ServiceToken> tokens = new HashSet<ServiceToken>();
         for (int count = random.nextInt(NUM_SERVICE_TOKENS); count >= 0; --count) {
+            final String name = SERVICE_TOKEN_NAME + random.nextInt();
             final byte[] data = new byte[8];
             random.nextBytes(data);
-            final ServiceToken token = new ServiceToken(ctx, "masterbound" + count, data, masterToken, null, false, null, cryptoContext);
+            final ServiceToken token = new ServiceToken(ctx, name, data, masterToken, null, false, null, cryptoContext);
```

Comment:
I think it still might be useful for debug to include the boundedness in the name.

Reply:
I originally removed the strings in order to avoid a miss in the unit tests because "masterbound1" ≠ "userbound1", but then changed the unit tests to not care, which is why name now includes a random just like the `getServiceTokens()` function earlier in the class. Going to leave it this way for consistency.